docs/filesets.md

```diff
@@ -23,10 +23,10 @@ required:
 
 ## File patterns
 
-The following patterns are supported:
+The following patterns are supported (the quotes are optional):
```

Comment:
Are the actual rules still like described above, "quotes can be omitted if the expression has no operators or function calls"? Should I still merge https://github.com/jj-vcs/jj/pull/5946/files as-is?